pkg/operator/ceph/csi/spec.go

```diff
@@ -486,13 +487,23 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 	}
 
 	if EnableRBD {
-		err = createCSIDriverInfo(ctx, clientset, RBDDriverName)
+		fsGroupPolicyForRBD, err := k8sutil.GetOperatorSetting(clientset, controllerutil.OperatorSettingConfigMapName, "CSI_RBD_FSGROUPPOLICY", "")
+		if err != nil {
+			// logging a warning and intentionally continuing with the default log level
+			logger.Warningf("failed to load CSI_RBD_FSGROUPPOLICY. %v", err)
```

Comment:
I think it would be good to indicate here what users can expect from the operator in this case. The operator continues, but what setting will it use? Will it disable the fsgrouppolicy?